src/sql/sqlops.proposed.d.ts

```diff
@@ -722,6 +722,11 @@ declare module 'sqlops' {
 		 */
 		isAutoResizable: boolean;
 
+		/**
+		 * Minimum height for editor component
+		 */
+		minimumEditorSize: number;
```

Comment:
Shouldn't it be `minimumHeight`? This is a more generic concept that could be up-leveled in the future, plus size would usually be a Dimension (height,width)